hawc/apps/assessment/forms.py

```diff
@@ -38,20 +38,22 @@ class AssessmentForm(forms.ModelForm):
     )
 
     class Meta:
-        exclude = (
```

Comment:
switched from exclude to fields (include), since it's easier to understand what's available.